sourcecode/npm/components/public/index.html

```diff
@@ -9,6 +9,7 @@
             content="Web site created using create-react-app"
         />
         <title>Edlib</title>
+        <link rel="icon" href="favicon.ico" />
```

Comment:
This would break if the page was accessed via a subpath like `/foo/bar.html`, as the browser would then try looking for the favicon at `/foo/favicon.ico`. In single-page applications such as these, every path is mapped to this index file. But as mentioned in the other comment, leaving `favicon.ico` in the public folder with no changes to the HTML is sufficient.